packages/gatsby/src/services/index.ts

```diff
@@ -0,0 +1,5 @@
+export { calculateDirtyQueries } from "./calculate-dirty-queries"
```

Comment:
Overall question, what's the reason to export everything from here and not categorize them with a statemachine? I guess we'll have a query-running state-machine,... So it might be better to sub organize them? I'm worried that this will get bloated

Reply:
That's exactly what we do in the state machine. It's exported as a services map that's part into the machine. This is temporary until all the services are in place. We  need to export them at the moment, as we're importing them in the "legacy" code too.